scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -62,3 +62,81 @@ def test_zero_norms_from_quaternion():
             ])
     with pytest.raises(ValueError):
         r = Rotation.from_quaternion(x)
+
+
+def test_as_dcm_single_1d_quaternion():
+    quat = [0, 0, 0, 1]
+    mat = Rotation.from_quaternion(quat).as_dcm()
+    # mat.shape == (3,3) due to 1d input
+    assert_array_almost_equal(mat, np.eye(3))
+
+
+def test_as_dcm_single_2d_quaternion():
+    quat = [[0, 0, 1, 1]]
+    mat = Rotation.from_quaternion(quat).as_dcm()
+    assert(mat.shape == (1, 3, 3))
```

Comment:
I don't know exact reasons 😄, but it seems to me that `assert_` from `numpy.testing` should be used instead of Python built-in assert. At least this is how it is done, I think it's better to follow.